src/main/java/org/jfrog/bamboo/context/GradleBuildContext.java

```diff
@@ -19,6 +19,7 @@
     public static final String BUILD_FILE_PARAM = "buildFile";
     public static final String USE_GRADLE_WRAPPER_PARAM = "useGradleWrapper";
     public static final String GRADLE_WRAPPER_LOCATION_PARAM = "gradleWrapperLocation";
+    public static final String PUBLISH_FORK_COUNT = "publishForkCount";
```

Comment:
To remain consistent with the other constants defined here, I suggest renaming *PUBLISH_FORK_COUNT* to *PUBLISH_FORK_COUNT_PARAM*

Reply:
agreed and done